sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -628,9 +682,13 @@ export interface GeneratedClientEntitiesRecognitionPiiOptionalParams
    */
   modelVersion?: string;
   /**
-   * (Optional) if set to true, response will contain input and document level statistics.
+   * (Optional) if set to true, response will contain request and document level statistics.
    */
   includeStatistics?: boolean;
+  /**
+   * (Optional) Specifies the method used to interpret string offsets.  Defaults to Text Elements (Graphemes) according to Unicode v8.0.0. For additional information see https://aka.ms/text-analytics-offsets
```

Comment:
we are default to 
utf16CodeUnit: iterates over UTF-16 Code Units, the default scheme for **Javascript**, Java, and .NET

Reply:
this is the description in the swagger file and this is a generated file that I can not change.